ArithmeticException in ImageUtils

Crash report from Google Play **new** for 2020.11.14-RC:

Currently only a single occurrence 

```
java.lang.RuntimeException: 
  at android.os.AsyncTask$4.done (AsyncTask.java:399)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:383)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:252)
  at java.util.concurrent.FutureTask.run (FutureTask.java:271)
  at android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:289)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:919)
Caused by: java.lang.ArithmeticException: 
  at cgeo.geocaching.utils.ImageUtils.readDownsampledImage (ImageUtils.java:251)
  at cgeo.geocaching.utils.ImageUtils.readScaleAndWriteImage (ImageUtils.java:209)
  at cgeo.geocaching.ImageSelectActivity$1.doInBackground (ImageSelectActivity.java:151)
  at cgeo.geocaching.ImageSelectActivity$1.doInBackground (ImageSelectActivity.java:144)
  at android.os.AsyncTask$3.call (AsyncTask.java:378)
  at java.util.concurrent.FutureTask.run (FutureTask.java:266)
```

--- a/main/src/cgeo/geocaching/utils/ImageUtils.java
+++ b/main/src/cgeo/geocaching/utils/ImageUtils.java
@@ -248,7 +248,7 @@ public final class ImageUtils {
         final BitmapFactory.Options sizeOnlyOptions = getBitmapSizeOptions(filePath);
         final int myMaxXY = Math.max(sizeOnlyOptions.outHeight, sizeOnlyOptions.outWidth);
         final int maxXY = Math.max(maxX <= 0 ? sizeOnlyOptions.outWidth : maxX, maxY <= 0 ? sizeOnlyOptions.outHeight : maxY);
-        final int sampleSize = myMaxXY / maxXY;
+        final int sampleSize = maxXY <= 0 ? 1 : myMaxXY / maxXY;
         final BitmapFactory.Options sampleOptions = new BitmapFactory.Options();
         if (sampleSize > 1) {
             sampleOptions.inSampleSize = sampleSize;

